integration-tests/README.md

```diff
@@ -0,0 +1,28 @@
+# Thoth-Station Integration Tests
+
+## Managining and Creating Overlays
+
+### About our overlays
+
+Overlays are used to manage and deploy our tests for different environments. One thing to note is that comprared to most overlays for different components within `Thoth-Station`, our prod overlays (Balrog and Smuag) also includes `ocp4-stage`. This labeling format is chosen because while the cronjob itself runs on anywhere, the notification job will run in the staging cluster (OCP4). We have opted for this architecture because to send these emails, the notification service must sit behind the VPN, such as our stage cluster.
+
+### Creating overlays
+
+There is also a configmap which is used in this deployment but is not deployed from this overlay
```

Comment:
It would be great if we could link the configmap present in slo-reporter.
and re-write the sentence with the link and straight subjective manner. 

something like: The thoth configmap is deployed via slo-reporter and also used with integration-tests.
please make sure to deploy the configmap separately if slo-reporter is not deployed along with integration-tests.